# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/diligent-engine/no-pip.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

diff --git a/DiligentCore/ThirdParty/dawn/CMakeLists.txt b/DiligentCore/ThirdParty/dawn/CMakeLists.txt
index e71a75a..bd02eed 100644
--- a/DiligentCore/ThirdParty/dawn/CMakeLists.txt
+++ b/DiligentCore/ThirdParty/dawn/CMakeLists.txt
@@ -2,18 +2,6 @@ cmake_minimum_required (VERSION 3.11)
 
 find_package(Python3 REQUIRED)
 
-set(JINJA2_INSTALL_CMD ${Python3_EXECUTABLE} -m pip install --upgrade jinja2>=3.0)
-
-if(${Python3_VERSION} VERSION_GREATER_EQUAL "3.12")
-    set(JINJA2_INSTALL_CMD ${JINJA2_INSTALL_CMD} --break-system-packages)
-endif()
-
-execute_process(COMMAND ${JINJA2_INSTALL_CMD}
-                RESULT_VARIABLE PYTHON_PIP_JINJIA_RESULT)
-if(NOT PYTHON_PIP_JINJIA_RESULT EQUAL "0")
-    message(FATAL_ERROR "Command '${JINJA2_INSTALL_CMD}' failed with error code ${PYTHON_PIP_JINJIA_RESULT}")
-endif()
-
 FetchContent_DeclareShallowGit(
     dawn
     GIT_REPOSITORY         https://dawn.googlesource.com/dawn
diff --git a/DiligentTools/RenderStateNotation/CMakeLists.txt b/DiligentTools/RenderStateNotation/CMakeLists.txt
index e89ef87..429f29c 100644
--- a/DiligentTools/RenderStateNotation/CMakeLists.txt
+++ b/DiligentTools/RenderStateNotation/CMakeLists.txt
@@ -26,26 +26,6 @@ file(COPY ../.clang-format DESTINATION "${RSN_PARSER_GENERATED_HEADERS_DIR}")
 
 find_package(Python3 REQUIRED)
 
-set(LIBCLANG_INSTALL_CMD ${Python3_EXECUTABLE} -m pip install libclang==16.0.6)
-set(JINJA2_INSTALL_CMD ${Python3_EXECUTABLE} -m pip install jinja2)
-
-if(${Python3_VERSION} VERSION_GREATER_EQUAL "3.12")
-    set(LIBCLANG_INSTALL_CMD ${LIBCLANG_INSTALL_CMD} --break-system-packages)
-    set(JINJA2_INSTALL_CMD ${JINJA2_INSTALL_CMD} --break-system-packages)
-endif()
-
-execute_process(COMMAND ${LIBCLANG_INSTALL_CMD}
-                RESULT_VARIABLE PYTHON_PIP_LIBCLANG_RESULT)
-if(NOT PYTHON_PIP_LIBCLANG_RESULT EQUAL "0")
-    message(FATAL_ERROR "Command '${LIBCLANG_INSTALL_CMD}' failed with error code ${PYTHON_PIP_LIBCLANG_RESULT}")
-endif()
-
-execute_process(COMMAND ${JINJA2_INSTALL_CMD}
-                RESULT_VARIABLE PYTHON_PIP_JINJIA_RESULT)
-if(NOT PYTHON_PIP_JINJIA_RESULT EQUAL "0")
-    message(FATAL_ERROR "Command '${JINJA2_INSTALL_CMD}' failed with error code ${PYTHON_PIP_JINJIA_RESULT}")
-endif()
-
 file(GLOB INCLUDE    include/*)
 file(GLOB INTERFACE  interface/*)
 file(GLOB SOURCE     src/*)
